deploy/examples/create-external-cluster-resources.py

```diff
@@ -617,10 +628,59 @@ def endpoint_dial(self, endpoint_str, ip_type, timeout=3, cert=None):
             ("-1"),
         )
 
+    def parse_config(self, config_file):
+        config = configparser.ConfigParser()
+        config.read(config_file)
+
+        configConfigurations = [
+            "verbose",
+            "ceph-conf",
+            "keyring",
+            "run-as-user",
+            "cluster-name",
+            "k8s-cluster-name",
+            "namespace",
+            "rgw-pool-prefix",
+            "restricted-auth-permission",
+            "v2-port-enable",
+            "format",
+            "output",
+            "cephfs-filesystem-name",
+            "cephfs-metadata-pool-name",
+            "cephfs-data-pool-name",
+            "rbd-data-pool-name",
+            "alias-rbd-data-pool-name",
+            "rgw-endpoint",
+            "rgw-tls-cert-path",
+            "rgw-skip-tls",
+            "monitoring-endpoint",
+            "monitoring-endpoint-port",
+            "skip-monitoring-endpoint",
+            "rbd-metadata-ec-pool-name",
+            "dry-run",
+            "rados-namespace",
+            "subvolume-group",
+            "rgw-realm-name",
+            "rgw-zone-name",
+            "rgw-zonegroup-name",
+            "topology-pool",
+            "topology-failure-domain-label",
+            "topology-failure-domain-values",
+            "upgrade",
+        ]
+
+        for config in configConfigurations:
+            config_value = config.get("Configurations", config, fallback=None)
+            self._arg_parser.config = self._arg_parser.config if self._arg_parser.config is not None else config_value
+
+        return config
+
     def __init__(self, arg_list=None):
         self.out_map = {}
         self._excluded_keys = set()
         self._arg_parser = self.gen_arg_parser(args_to_parse=arg_list)
+        # Parse configuration file
+        self.config = self.parse_config(self._arg_parser.config)
```

Comment:
Comment seems redundant with `parse_config()` method name

```suggestion
        self.config = self.parse_config(self._arg_parser.config)
```

Reply:
Updated